src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/RespondExtensionCallbackHandler.java

```diff
@@ -107,7 +108,7 @@ private SubmittedCallbackResponse buildConfirmation(CallbackParams callbackParam
         Map<String, Object> data = callbackParams.getRequest().getCaseDetails().getData();
         LocalDateTime responseDeadline = mapToDateTime(data, RESPONSE_DEADLINE);
 
-        String claimNumber = "TBC";
+        String claimNumber = mapper.convertValue(data.get(LEGACY_CASE_REFERENCE), String.class);
```

Comment:
do we need to map this?

Reply:
yes, we have to display on confirmation screen.